ivy.el

```diff
@@ -4426,6 +4426,20 @@ EVENT gives the mouse position."
                                 (expand-file-name "doc/ivy-help.org"))))
   "The file for `ivy-help'.")
 
+(defun ivy-word-at-point-dwim ()
+  "Extracts a meaningful part of the current buffer.
```

Comment:
The elevator pitch part of that docstring sucks. If anyone has a better proposal...